src/app/modules/registration/components/edit-images/edit-images.component.html

```diff
@@ -19,6 +19,42 @@
       [rows]="2"
       placeholder="{{ pictureCommentPlaceholder() | translate }}"
     ></app-text-comment>
+    <ion-accordion-group>
+      <ion-accordion>
+        <ion-item slot="header" class="accordion-button">
+          <ion-label>{{ "REGISTRATION.SNOW.SNOW_PROFILE.STRAT_PROFILE.SHOW_MORE" | translate }}</ion-label>
```

Comment:
Det kan hende vi burde laget en egen oversettelse for dette, men enn så lenge går det sikkert greit. Enten laget en egen eller flytta den vi har nå ut fra snow profile for å vise at den brukes flere steder.
Men som sagt, går greit for meg å ta det inn sånn her også.